docs/source/markdown/podman-info.1.md

```diff
@@ -71,6 +71,9 @@ host:
       commit: 3e425f80a8c931f88e6d94a8c831b9d5aa481657
       spec: 1.0.1-dev
   os: linux
+  remoteSocket:
+    exists: false
```

Comment:
I think this is inaccurate: when I test it on my system, I don't get an `exists` line at all. Could it be the `omitempty` below? I'd like to see this reflect reality, either by removing the `omitempty` or removing the `false` here.